arthur_bench/models/models.py

```diff
@@ -275,9 +284,36 @@ class SummaryItem(BaseModel):
 
     id: UUID
     name: str
-    avg_score: float
+    avg_score: Optional[float]
     histogram: List[HistogramItem]
 
+    @validator("histogram")
+    def either_lowhigh_or_category(cls, v):
```

Comment:
I think there are some edge cases that are not caught by this validator. For example,

`HistogramItem(low=0.0, high=None, category='negative')`

would be valid according to this check